app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_api_v1_user!, only: %i[index create]
+
+      def index
+        render_paginated Rent.filter(params.slice(:user_id, :book_id))
```

Comment:
you forgot to leave the includes? :D

Reply:
Which includes? Maybe you talk about the ones in the models

Reply:
the ones we discussed yesterday to avoid a lot of queries

Reply:
Oh you're right :D My bad.